pytorch_lightning/loops/epoch/prediction_epoch_loop.py

```diff
@@ -89,7 +89,8 @@ def advance(  # type: ignore[override]
             num_dataloaders: the total number of dataloaders
             return_predictions: whether to return the obtained predictions
         """
-        batch_idx, batch = next(dataloader_iter)
+        with self.trainer.profiler.profile(f"[{self.__class__.__name__}].pred_dataloader_next"):
+            batch_idx, batch = next(dataloader_iter)
```

Comment:
`prediction_epoch_loop` doesn't seem to use data fetchers, should I just leave this as is?

Reply:
Yes